Documentation/getting_started/2_understanding_the_code.md

```diff
@@ -72,19 +82,22 @@ protected override void Update(GameTime gameTime)
     base.Update(gameTime);
 }
 ```
-This method is called multiple times per second(by default 30), and is used to update your game state (checking for collisions, gathering input, playing audio, etc.)
+
+This method is called multiple times per second, and is used to update your game state (checking for collisions, gathering input, playing audio, etc.).
 
 **Draw Method**
+
 ```csharp
 protected override void Draw(GameTime gameTime)
 {
     graphics.GraphicsDevice.Clear(Color.CornflowerBlue);
 
     //TODO: Add your drawing code here
-    
+
     base.Draw(gameTime);
 }
 ```
-Similar to Update method, it is also called multiple times per second. The reason why Update and Draw methods are split is so that even if you have to do some heavy calculations(ie. path finding), you won't slow down your game fps.
+
+Similar to Update method, it is also called multiple times per second. Unlike the update method, this method is used for drawing stuff onto the screen.
```

Comment:
Similar to **the** Update method
Unlike the update method -> I'd drop this part